Source/WebCore/css/CSSComputedStyleDeclaration.cpp

```diff
@@ -643,10 +643,14 @@ static Ref<CSSValue> computedTranslate(RenderObject* renderer, const RenderStyle
     auto list = CSSValueList::createSpaceSeparated();
     list->append(zoomAdjustedPixelValueForLength(translate->x(), style));
 
-    if (!translate->y().isZero() || !translate->z().isZero())
+    auto showLength = [&](Length length) -> bool {
```

Comment:
"show" here isn't quite the right word. Maybe "include". Also `const Length&`

Also does not need `&` capture. or `-> bool`